suite-native/module-send/src/hooks/useAddressValidationAlerts.tsx

```diff
@@ -0,0 +1,138 @@
+import { useCallback, useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { useRoute, RouteProp } from '@react-navigation/native';
+import { checkAddressCheckSum, toChecksumAddress } from 'web3-utils';
```

Comment:
fyi: `checkAddressCheckSum` is marked deprecated

Reply:
I will let the desktop trends team know. [The web3-utils is their dependency](https://github.com/trezor/trezor-suite/blob/1b02b0cd14a4d2ece15bbd5b6019cc04b031cb8f/scripts/list-outdated-dependencies/trends-dependencies.txt#L7). So I will leave this up to them how to resolve it.